tests/apis/prv_api/test_status_changes.py

```diff
@@ -162,6 +162,8 @@ def test_device_list_basic_time(
     data = response.data["data"]["status_changes"]
     assert len(data) == 2
 
+    for el in data:
```

Comment:
Je comprends pas, pourquoi tu pop ? pourquoi tu ajouterais pas ce champs dans les expected_event_device ?

Reply:
Yes you should modify `expected_event_device1` instead